src/test/java/calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,96 @@
+package calculator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatExceptionOfType;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+
+public class StringCalculatorTest {
+
+  private StringCalculator stringCalculator;
+
+  @BeforeEach
+  void init() {
+    stringCalculator = new StringCalculator();
+  }
+
+  @Test
+  @DisplayName("덧셈 테스트")
+  void addTest() {
```

Comment:
`StringCalculatorTest`란 이름이 `StringCalculator` 클래스의 기능들을 테스트한다는 의미를 내포하고 있으니
테스트 메서드마다 `-Test`와 같은 접미사를 붙이지 않아도 될 것 같은데, 용훈님께서는 어떻게 생각하시나요?
```suggestion
  void add() {
```